### DISCUSSION TITLE
Streaming of the frigate analysis image to HA?

### DISCUSSION THREAD

**Discussion by mickeynygaard** [blakeblackshear/frigate#1687] [Ideas]
If this should be posted as a suggestion to whomever has created the HA integration instead, please let me know and I'll move the question.

Right now the streams I have via the HA integration are both high-res and runs at more FPS than I would like, seeing as a lot of data has to be transferred and I may not always have the best bandwidth available.

When I go to the frigate UI and click a camera I get a live feed of the camera running at 5 FPS with all the analysis of movement going on. I would actually love it if I could have this as my camera within HA, as it seems to be updating faster, runs with less bandwidth and allows me to see what the frigate analysis sees.

Thanks for a cool product...for most part the motion detection is spot on and I enjoy the simple yet powerful UI.

**Comment by blakeblackshear** [comment]
You can create custom camera entities in home assistant using the mjpeg endpoint [here](https://blakeblackshear.github.io/frigate/usage/api#apicamera_name). You can use the parameters to customize it to your liking. I don't generally recommend it because mjpeg is very bandwidth heavy and generates extra load on frigate.

**Comment by mickeynygaard** [comment]
Thanks for the reply mate, and again, thank you for a cool product!
The feed I get in my Home Assistant seems to be delayed quite a bit compared to what I see in Frigate...everything is running on the same PC in different dockers, so I can't quite figure why the stream would be delayed...I also have no need for 30+ FPS when streaming but I would like my recordings and clips to be in 30 FPS...not sure if I'm missing something obvious here?
